### PR TITLE
fixing outdated metis link

### PR DESCRIPTION
**Pull request by mlawsonca**

### CODE REVIEW

File: var/spack/repos/builtin/packages/metis/package.py [hunk 1]
```diff
@@ -22,7 +22,8 @@ class Metis(Package):
     #
 
     homepage = "https://github.com/scivision/METIS/"
-    url      = "https://github.com/scivision/METIS/raw/master/metis-5.1.0.tar.gz"
+    #url      = "https://github.com/scivision/METIS/raw/master/metis-5.1.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/metis/package.py]: Delete this line

**Review by adamjstewart — CHANGES_REQUESTED** [review]
I'm not sure what happened to the download link. It looks like the GitHub page now only has a newer release, so maybe they simply removed the last release? http://glaros.dtc.umn.edu/gkhome/metis/metis/download has the 5.1.0 release as well, while http://glaros.dtc.umn.edu/gkhome/fsroot/sw/metis/OLD has older versions. I wonder if we should consider that website more stable than the GitHub if they decide to keep removing old releases.

**Comment by adamjstewart** [comment]
Opened https://github.com/scivision/METIS/issues/3 to investigate more.

**Comment by adamjstewart** [comment]
It looks like the current URL was introduced by @cyrush in #13709. From what I can tell, this GitHub repo is an unofficial mirror and only contains the latest version. Now that we have Spack mirrors for things, I think it would be better to switch back to the old `url`/`list_url`. @cyrush thoughts?

File: var/spack/repos/builtin/packages/metis/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/metis/package.py]: Also, can you switch to:
```python
    homepage = "http://glaros.dtc.umn.edu/gkhome/metis/metis/overview"
    url      = "http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz"
    list_url = "http://glaros.dtc.umn.edu/gkhome/fsroot/sw/metis/OLD"
```
Then you can remove the URL from 4.0.3.

**Comment by cyrush** [comment]
@adamjstewart I also vote for using the official urls + spack mirror.

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Thanks @mlawsonca!

**Pull request merged**